tests/test_alarm.py

```diff
@@ -48,9 +48,11 @@ def test_duration_and_repeat() -> None:
 
 def test_display_required_fields() -> None:
     """Test required fields for action DISPLAY."""
-    with pytest.raises(ValidationError):
+    with pytest.raises(ValidationError) as e:
         Alarm(action="DISPLAY", trigger=datetime.timedelta(minutes=-5))
 
+    assert "Description value is required for action DISPLAY" in e.value.raw_errors[0].exc.args[0]
+
```

Comment:
I believe you can use the built in matcher for this:

```suggestion
    with pytest.raises(ValidationError, match="Description value is required for action DISPLAY"):
        Alarm(action="DISPLAY", trigger=datetime.timedelta(minutes=-5))
```

(here and below)

Reply:
Well, thats way better. Thanks for the hint.